src/inspector/view/qml/MuseScore/Inspector/notation/sectionbreaks/SectionBreakSettings.qml

```diff
@@ -71,6 +73,7 @@ Column {
     }
 
     PropertyCheckBox {
+        //: Watch out for string length and inspector width
```

Comment:
Not sure we really need these warnings. Qt will wrap the strings if they are too long. If they don't wrap then it's a bug. It would mean we need to set the [`wrapMode` property](https://doc.qt.io/qt-6/qml-qtquick-text.html#wrapMode-prop) somewhere.

Reply:
yep